client/console/src/main/java/org/apache/syncope/client/console/panels/UserDirectoryPanel.java

```diff
@@ -342,6 +343,23 @@ public void onClick(final AjaxRequestTarget target, final UserTO ignore) {
             }, ActionType.MANAGE_ACCOUNTS,
                     String.format("%s,%s,%s", StandardEntitlement.USER_READ, StandardEntitlement.USER_UPDATE,
                             StandardEntitlement.RESOURCE_GET_CONNOBJECT));
+
+            panel.add(new ActionLink<UserTO>() {
+                private static final long serialVersionUID = 8011039414597736111L;
+
+                @Override
+                public void onClick(final AjaxRequestTarget target, final UserTO ignore) {
+                    model.setObject(UserRestClient.class.cast(restClient).read(model.getObject().getKey()));
+                    MergeLinkedAccountsWizardBuilder builder =
+                        new MergeLinkedAccountsWizardBuilder(model, pageRef, UserDirectoryPanel.this, modal);
```

Comment:
I notice that once the wizard is completed the table is not refreshed (e.g. the deleted user is still visibile): maybe @fmartelli can tell you more.